common/firmware/spi/spi_comms.cpp

```diff
@@ -25,12 +26,24 @@ using namespace spi;
  * Public Functions
  */
 
-Spi::Spi() { SPI2_init(); }
+Spi::Spi() {
+    // get SPI handle from spi.c
+    void* tmp;
```

Comment:
Same as above. I don't get why we would use `void*`.

Reply:
[2e47656](https://github.com/Opentrons/ot3-firmware/pull/145/commits/2e476562682799348290143c475adbb72ead3dc9) makes spi.c and spi_comms.cpp less complicated.